File: ClinicApi/Models/Booking/UpdateBookingModel.cs
using ClinicApi.Models.Document;
using System.Collections.Generic;

namespace ClinicApi.Models.Booking
{
    public class UpdateBookingModel: BookingModel
    {
        public int ClinicId { get; set; }
        public int ClinicianId { get; set; }
        public IEnumerable<DocumentModel> DeletedDocuments { get; set; }
    }
}

File: ClinicWeb/src/app/core/models/booking/update-booking.model.ts
import { DocumentModel } from './document.model';
import { Stage } from '..';

export class UpdateBookingModel {
    id: number;
    name: string;
    documents: DocumentModel[];
    clinicId: number;
    clinicianId: number;
    newFiles: any[] = [];
    deletedDocuments: DocumentModel[] = [];
    rate: number;
    HeartRate: number;
    Weight: number;
    Height: number;
    PatientDescription: string;
    stage: Stage;
}
